docs/metrics.asciidoc

```diff
@@ -10,7 +10,7 @@ The .NET agent tracks certain system and application metrics.
 Some of them have built-in visualizations and some can only be visualized with custom Kibana dashboards.
 
 These metrics will be sent regularly to the APM Server and from there to Elasticsearch.
-You can adjust the interval with the setting <<config-metrics-interval>>.
+You can adjust the interval with the setting <<config-metrics-interval,`MetricsInterval`>>.
```

Comment:
Didn't we already have this [here](https://github.com/elastic/apm-agent-dotnet/pull/369#discussion_r301045128)? 

This time the build does not fail...

Reply:
At https://github.com/elastic/apm-agent-dotnet/pull/369#discussion_r301139742 the problematic part was `configuration.asciidoc#` - it cause docs build failure.

Reply:
The reason I've added

> ,\`MetricsInterval\`

is because without it (i.e., just

> \<\<config-metrics-interval\>\>

Renders as (see https://www.elastic.co/guide/en/apm/agent/dotnet/current/metrics.html)

> `MetricsInterval ( )`

Because section title has `(added[1.0.0-beta1])` in it

> [[config-metrics-interval]]
==== `MetricsInterval` (added[1.0.0-beta1])

Reply:
Oh, I see. Strange that it's not rendered properly. Yeah, fine by me.